content/en/docs/getting-started/upgrade.md

```diff
@@ -39,8 +44,7 @@ apt-get --only-upgrade install falco
 ### CentOS/RHEL/Fedora/Amazon Linux {#centos-rhel}
 
 {{% pageinfo color="warning" %}}
-If you configured the `yum` repository by having followed the instructions for Falco 0.27.0 or older, 
-you may need to update the repository URL:
+If you configured the `yum` repository by having followed the instructions for Falco 0.27.0 or older, you may need to update the repository URL, otherwise, **fill free to ignore this message**
```

Comment:
```suggestion
If you configured the `yum` repository by having followed the instructions for Falco 0.27.0 or older, you may need to update the repository URL, otherwise, **feel free to ignore this message**
```